Scripts/StructPackVerifier.py

```diff
@@ -10,6 +10,18 @@
 logger = logging.getLogger()
 logger.setLevel(logging.WARNING)
 
+# These defines are temporarily defined since python3-clang doesn't yet support these.
```

Comment:
Why do we need these defines despite not using them?

Reply:
We need to define these globally ourselves since python3-clang will pull them in to their global namespace. Otherwise we get an assert when we query the CXCursor kind through its `kind` function.
Python3-clang is missing a couple hundred CursorKind values sadly.